catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -157,21 +116,14 @@ public boolean getDisableCnCheck() {
 
     public void setDisableCnCheck(boolean disableCnCheck) {
         this.disableCnCheck = disableCnCheck;
-
     }
 
     public Map<String, String> getMetacardCswMappings() {
         Map<String, String> newMap = new HashMap<>();
-        for (Entry<String, String> entry : metacardCswMappings.entrySet()) {
-            newMap.put(entry.getValue(), entry.getKey());
-        }
+        newMap.putAll(metacardCswMappings);
```

Comment:
How is the behavior of `putAll()` different from using the copy constructor? Besides, the keys and values in this map are strings (immutable), so a shallow copy won't matter.